src/devices/bosch.ts

```diff
@@ -350,6 +451,23 @@ const tzLocal = {
 
 
 const fzLocal = {
+    ias_siren: {
+        cluster: 'ssIasZone',
+        type: 'commandStatusChangeNotification',
+        convert: async (model, msg, publish, options, meta) => {
+            const zoneStatus = msg.data.zonestatus;
+            await msg.endpoint.command('ssIasZone', 'boschTestTamper', {data: 2}, {manufacturerCode: 0x1209});
```

Comment:
Can you add a comment why this is needed?

Reply:
@smarts4home can you still check this comment?

Reply:
Siren send this periodically, This is one byte of data, bit by bit with status information about siren

Reply:
But why is it needed to send a command to the device when it sends a `commandStatusChangeNotification`? Does something not work otherwise?